server.py

```diff
@@ -253,8 +253,9 @@ def get_app(server):
 @click.command()
 @click.option("-m", "--map-name", default="maps/test_winner.json",
               help="Name of the played map.")
-def main(map_name):
-    server = Server(map_name)
+@click.option("-p", "--players", default=8, help="Number of players")
```

Comment:
Když řekneš Clicku, že chceš číslo, nemusíš pak převádět v kódu. Dělá se to přes [`type=int`](https://click.palletsprojects.com/en/7.x/parameters/#parameter-types).